maybe/docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+version: '3.7'
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: maybe_web_1
+      APP_PORT: 3063
+
+  web:
+    image: ghcr.io/maybe-finance/maybe@sha256:f1d1d0d00017e009dd805a184fe5e2aaa663642f6e5bd8c22be5d35daa85c2e6
+    restart: on-failure
+    stop_grace_period: 1m
+    volumes:
+      - ${APP_DATA_DIR}/data/rails:/rails/storage
+    env_file:
+      - ${APP_DATA_DIR}/.env
```

Comment:
If I understand correctly, this `.env` file is here to allow a user to add their synth api key and have it be persisted across app updates, correct? This is a tricky one.

Ideally, apps will let users configure everything through the UI, but it seems Maybe doesn't support this for the api key yet. Hopefully, Maybe will add a feature to input an API key through the UI in the future (we should submit an Issue requesting this). This is likely since the self-hosted version of the app is currently very minimal and still under development, with most features labeled as "coming soon."

There is no inbuilt way in umbrelOS right now to add environment variables to apps through the umbrelOS UI so, no matter what, a user would need to access the terminal to add the api key (can be through advanced settings in umbrelOS 1.1+). Perhaps we should just leave it to advanced users who need the api key to add the env var `SYNTH_API_KEY: <their-api-key>` to Maybe's compose file and then restart the app from the homescreen. This would be an interim measure until Maybe hopefully adds the ability to do this through the UI.

This would be instead of going with the `env_file` method, which has the following drawbacks:
- If there is no `.env` committed to this repo then the app will fail to install because Docker will error on not being able to find it. We could get around this by committing an empty .env file to this repo, but...
- The `.env` would not be one of the whitelisted files that will be pulled down during app updates: https://github.com/getumbrel/umbrel/blob/5a042334e36e65128058f512c29e7af422f4be0c/packages/umbreld/source/modules/apps/legacy-compat/app-script#L594-L595. So we'd saddle ourselves with needing to add hooks in the future if we needed to modify this file for all users.

Reply:
Maybe I made a mistake while testing it, as it seemed to work OK without it, but I'll add an empty file - the main thing I wanted to avoid was it being overwritten when the app updated.

Most people probably won't need it as all their accounts will be in their local currency.